### PR TITLE
README note on Scorecard tests

### PR DESCRIPTION
**Pull request by razo7**
#### Why we need this PR

Include how to easily use scorecard testing to validate the operator bundle directory. Follow up to #141 

#### Changes made

- Add a new subsection in README for how to use Scorecard testing.

### CODE REVIEW

**Comment by razo7** [comment]
/override ci/prow/4.12-test
/override ci/prow/4.13-test
/override ci/prow/4.14-test
/override ci/prow/4.15-test
/override ci/prow/4.16-test

Overriding the required tests that are skipped in docs only changes

File: README.md [hunk 1]
```diff
@@ -235,6 +235,10 @@ Run `make test`
 3. Export the operator installed namespace (e.g., *openshift-workload-availability*) before running the e2e test:
  `export OPERATOR_NS=openshift-workload-availability && make test-e2e`
 
+### Run Scorecard tests
+
+  Run `make test-scorecard` on a running Kubernetes cluster to statically validate the operator bundle directory using [Scorecard](https://sdk.operatorframework.io/docs/testing-operators/scorecard/).
```
Comment (slintes) [inline README.md:240]: how do you run that command "on a cluster"?

**Pull request merged**